### PR TITLE
View products front end

### PR DESCRIPTION
**Pull request by AndrewRobins51413**

### CODE REVIEW

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Please let me know when this is up-to-date

File: package.json [hunk 1]
```diff
@@ -86,11 +86,13 @@
     }
   },
   "dependencies": {
+    "bootstrap": "4.4.1",
     "dotenv": "8.2.0",
     "express": "4.17.1",
     "express-session": "1.17.0",
     "pg": "7.17.1",
     "react": "16.12.0",
+    "react-bootstrap": "1.0.0-beta.16",
```
Comment (thebearingedge) [inline package.json]: You can delete `react-bootstrap` and `bootstrap` from your `package.json`.

File: client/components/product-list.jsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+            ))
+          }
+        </div>
+
+        {/* <div className="menu-heading">
+                    <img className="card-img-top" src="https://github.com/Learning-Fuze/wicked-sales-js-instructions/blob/master/features/images/view-products-front-end.png?raw=true" alt="..."></img>
+                    <div className="card-body">
+                        <p className="card-text">Buy The Things!</p>
+                    </div>
+                </div> */}
```
Comment (thebearingedge) [inline client/components/product-list.jsx]: Please delete commented out code.

File: client/components/product-list.jsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+// import ReactDOM from "react-dom"
```
Comment (thebearingedge) [inline client/components/product-list.jsx]: You can delete this import if you don't need it.

File: client/components/app.jsx [hunk 4]
```diff
@@ -9,17 +11,10 @@ export default class App extends React.Component {
...
   render() {
-    return this.state.isLoading
-      ? <h1>Testing connections...</h1>
-      : <h1>{this.state.message}</h1>;
+    return <ProductList/>;
+
   }
 }
+
+// <Header/>,
```
Comment (thebearingedge) [inline client/components/app.jsx]: Please remove commented out snips.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

**Comment by AndrewRobins51413** [comment]
Screen Shot 2020-01-15 at 4 05 51 PM
Screen Shot 2020-01-15 at 4 05 51 PM

File: client/components/app.jsx (hunk 4 above)
Comment (AndrewRobins51413) [reply client/components/app.jsx]: I was not able to render multiple scripts through app.jsx. Not crucial for progress at the moment, but something I would like to know how to do.

File: package.json (hunk 1 above)
Comment (AndrewRobins51413) [reply package.json]: done

File: client/components/product-list.jsx (hunk 2 above)
Comment (AndrewRobins51413) [reply client/components/product-list.jsx]: My first effort did not involve bootstrap as much as I later realized it should have. I'll delete the non-bootstrap code when I have the bootstrap up and running.

File: client/components/product-list.jsx (hunk 3 above)
Comment (AndrewRobins51413) [reply client/components/product-list.jsx]: done

**Comment by AndrewRobins51413** [comment]
Screen Shot 2020-01-16 at 9 53 34 AM

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**